lib/csv/parser.rb

```diff
@@ -376,11 +380,23 @@ def prepare_regexp
       else
         @row_ends = nil
       end
+      @escaped_quotes = Regexp.new("(".encode(@encoding) +
+                           escaped_escape_character +
+                           escaped_quote_character +
+                           ")".encode(@encoding) +
+                           "+".encode(@encoding))
```

Comment:
We can't use multiple characters match to support stream.
See https://travis-ci.org/ruby/csv/jobs/483899256

Reply:
Indentation is broken.

Reply:
Fix to support stream when backslash_quote. https://github.com/ruby/csv/pull/74/commits/5919c093333f92b8061df2923342a6efbdc7fb8e
and Add `benchmark:parse_liberal_parsing` https://github.com/ruby/csv/pull/74/commits/16562b1e0972eaaf88053d3ca934933783c6bc8e